pkg/backend/net/net.go

```diff
@@ -370,7 +383,7 @@ func (b Backend) Info(ctx context.Context, request types.InfoRequest) ([]types.I
 	u = carbonapiV2InfoEncoder(u, metric)
 	request.Trace.AddMarshal(t0)
 
-	_, resp, err := b.call(ctx, request.Trace, u)
+	_, resp, err := b.call(ctx, request.Trace, u, "info")
```

Comment:
Why don't we keep `trace.AddLimiter` and use the time stored there after `b.call` to observe the time on histogram?
This way, we omit this `request string` argument which looks a bit weird IMHO.

Reply:
`trace` will go away, no need to keep it. Also, it averages stats which is wrong.